recipe/meta.yaml

```diff
@@ -23,6 +39,11 @@ requirements:
         - ocl-icd  # [linux]
         - khronos-opencl-icd-loader  # [win]
         - mingwpy  # [win]
+{% if osx_variant == "pocl" %}
+        - osx-pocl-opencl
+        - pocl
```

Comment:
Ah! That makes more sense. Clever design.

Leave it to Apple to buck the curve. Though I'm sure they have their own reasons for it.